rules/windows/persistence_dontexpirepasswd_account.toml

```diff
@@ -75,11 +75,17 @@ tags = [
     "Data Source: System",
 ]
 timestamp_override = "event.ingested"
-type = "query"
+type = "eql"
 
 query = '''
-event.action:"modified-user-account" and winlog.api:"wineventlog" and event.code:"4738" and
-  message:"'Don't Expire Password' - Enabled" and not user.id:"S-1-5-18"
```

Comment:
Here is some context on why we had to use the message field here: https://github.com/elastic/detection-rules/pull/1790#discussion_r819921435

PR that fixed the issue and enables us to use `winlog.event_data.NewUACList`: https://github.com/elastic/integrations/pull/8361